kotlin-runtime/build.gradle

```diff
@@ -10,7 +10,8 @@ dependencies {
 
     compileOnly libs.kotlinx.coroutines.core
     compileOnly libs.kotlinx.coroutines.reactive
-    compileOnly(libs.graal.svm)
+    compileOnly libs.kotlinx.coroutines.reactor
```

Comment:
is this correct? i think asFlow would be missing without this dependency, but the converter can still be called if kotlinx-coroutines and reactor-core are on the classpath

Reply:
Dependencies are required to be added by the user

Reply:
ok